pkg/miniogw/config.go

```diff
@@ -35,6 +35,15 @@ type RSConfig struct {
 	MaxThreshold     int `help:"the largest amount of pieces to encode to. n." default:"95"`
 }
 
+// NodeSelectionConfig is a configuration struct that keeps information about
```

Comment:
maybe specify in this comment that these are minimum values for nodes we want to select